pkg/daemon/daemon.go

```diff
@@ -17,41 +17,43 @@ const (
 	packetLength       = 10
 )
 
+// Packet defines a packet type
 type Packet struct {
 	PublicKey string
 	IP        string
 	T         int64
 }
 
+// Daemon provides configuration parameters for a
+// skywire-peering-daemon.
 type Daemon struct {
-	PublicKey  string
-	RemoteAddr string
-	PacketMap  map[string]string
-	DoneCh     chan error
-	PacketCh   chan []byte
-	Logger     *logging.Logger
-	NamedPipe  string
+	PublicKey string
+	localAddr string
+	PacketMap map[string]string
+	DoneCh    chan error
+	PacketCh  chan []byte
+	Logger    *logging.Logger
+	NamedPipe string
 }
 
 // NewDaemon returns a Daemon type
 func NewDaemon(pubKey, rAddr, namedPipe string) *Daemon {
 	return &Daemon{
-		PublicKey:  pubKey,
-		RemoteAddr: rAddr,
-		PacketMap:  make(map[string]string),
-		DoneCh:     make(chan error),
-		PacketCh:   make(chan []byte, packetLength),
-		Logger:     logger("SPD"),
-		NamedPipe:  namedPipe,
+		PublicKey: pubKey,
+		localAddr: rAddr,
```

Comment:
If this is indeed the local address, then `rAddr` as variable name is confusing.